### ISSUE TITLE
Question about GDAL.gdalbuildvrt: save more than one raster

### ISSUE THREAD

**Issue by Rapsodia86** [JuliaGeo/GDAL.jl#112]
Hello, 
I am working to rewrite my python script to Julia.
Here is the python part with I do struggle in the conversion process:
###in python####
    outvrt = "/vsimem/stacked.vrt"   
    outds  = gdal.BuildVRT(outvrt, tifs, separate=True)
####
while tifs is a list with different filenames created using append function
Out[32]: 
['U:/data/VI\\VNDVI_20190303.tif',
 'U:/data/VI\\VNDVI_20190317.tif',
 'U:/data/VI\\VNDVI_20190321.tif']
So, it creates me a VRT with many rasters according to the "tifs" list. The option "separate-TRUE" does this.
#############
Now, in Julia, I have created a vector (called tifs) with the same filenames (80 in total). Also, not being sure [I am very new in the Julia world] if vector is a proper type, I have reshaped it to an array as an alternative input. 
And so I have used:
#####in Julia
tifs[1:3]
3-element Array{String,1}:
 "U:/Data/VI/\\VNDVI_20190303.tif"
 "U:/Data/VI/\\VNDVI_20190317.tif"
 "U:/Data/VI/\\VNDVI_20190321"

options = GDAL.gdalbuildvrtoptionsnew([], C_NULL)
ds_buildvrt = GDAL.gdalbuildvrt("U:/data/stacked3.vrt", 80, C_NULL, tifs, options, C_NULL)
GDAL.gdalbuildvrtoptionsfree(options)
GDAL.gdalclose(ds_buildvrt)

The VRT is created but it only has one file according to the number from second parameter.
How I can save all rasters altogether? Is there a way to apply "separate = true" option to save all of the files from the filelist? Or, should be the filelist constructed differently? 

Julia version: v"1.5.3"
pkgs["GDAL"] v"1.2.0"
pkgs["ArchGDAL"] v"0.5.2"

Thank you for the help and suggestions!

**Comment by visr** [comment]
Hi, welcome!

> if vector is a proper type, I have reshaped it to an array as an alternative input

Vector of String should be good here. A Vector is a 1D Array, as shown here in julia 1.6:

```julia-repl
julia> Array{String, 1}
Vector{String} (alias for Array{String, 1})
```

> The VRT is created but it only has one file according to the number from second parameter.

So you expect 80 files in the VRT but only get 1? Only the first one in the list?

Could it be that the backslashes in the string are the problem?

In Python your mixed slash path is fine:
```
>>> 'U:/data/VI\VNDVI_20190303.tif'
'U:/data/VI\\VNDVI_20190303.tif'
```

Julia is a bit more strict:
```julia-repl
julia> "U:/data/VI\VNDVI_20190303.tif"
ERROR: syntax: invalid escape sequence
```

You need to either use forward slashes, or escape your backslashes with a backslash:
```julia-repl
julia> "U:/data/VI\\VNDVI_20190303.tif"
"U:/data/VI\\VNDVI_20190303.tif"
julia> println(ans)
U:/data/VI\VNDVI_20190303.tif
```

**Comment by visr** [comment]
Ah but to answer your question about -separate, you can pass that to the options you are currently giving [] to.

Similar to
https://github.com/JuliaGeo/GDAL.jl/blob/aa7ab211223e4ea5f515a602b9606610d84983b6/test/gdal_utils.jl#L9

**Comment by Rapsodia86** [comment]
Greetings!
Thank you!

I noticed in the python script, while building filelist as 
tifs = glob.glob(vis_dir2 + '*.tif')
in the "vis_dir2" definition I had + '/' +, instead of + "/" +;
while in Julia
tifs = glob("*.tif",hay_dir) 
and in "hay_dir" definition I did not have forward slashes at the end.
Both mistakes in codes resulted in having two backslashes. Codes are corrected now to have only forward slashes. I should be more careful, thanks for pointing this out.

In my post, I shortened up those directories and accidently removed the first backslash of those two, as well as".tif" from the third file. That created even worse mess. Apologies. 

In terms of the VRT  "-separate" option was exactly what I needed! The mixed slash path was not a problem (although I have it consistent now).

I am very impressed how quicker the VRT of 80 images was built in Julia than in python. That is really something.
Thank you very much again!
